helm/extensions.bzl

```diff
@@ -44,8 +49,24 @@ def _helm_impl(ctx):
             toolchain_config["helm_url_templates"] = toolchain_option.helm_url_templates
             toolchain_config["plugins"] = toolchain_option.plugins
 
+        for chart in module.tags.import_repository:
```

Comment:
Why is this restricted to only the root module? I think I understand it for registering toolchains but I would like to thing fetching a helm chart is fine.